tests/src/test/scala/cats/tests/CokleisliTests.scala

```diff
@@ -11,6 +11,9 @@ import org.scalacheck.Arbitrary
 
 class CokleisliTests extends SlowCatsSuite {
 
+  implicit override val generatorDrivenConfig: PropertyCheckConfiguration =
+    slowCheckConfiguration.copy(sizeRange = slowCheckConfiguration.sizeRange.min(5))
```

Comment:
20 sounds safe enough to me. 1m is a big difference for people who wants to work on cokleisli.